evalml/automl/automl_search.py

```diff
@@ -983,20 +1009,28 @@ def search(self, show_iteration_plot=True):
             current_batch_pipeline_scores = full_rankings[current_batch_idx][
                 "validation_score"
             ]
+
             if (
                 len(current_batch_pipeline_scores)
                 and current_batch_pipeline_scores.isna().all()
             ):
                 raise AutoMLSearchException(
                     f"All pipelines in the current AutoML batch produced a score of np.nan on the primary objective {self.objective}."
                 )
+            if len(pipeline_times) > 0:
+                pipeline_times["Total time of batch"] = time_elapsed(start_batch_time)
+                batch_times[self._get_batch_number()] = pipeline_times
 
         self.search_duration = time.time() - self._start
         elapsed_time = time_elapsed(self._start)
         desc = f"\nSearch finished after {elapsed_time}"
         desc = desc.ljust(self._MAX_NAME_LEN)
         self.logger.info(desc)
 
+        if timing == "log":
+            log_title(self.logger, "Batch Time Stats")
+            log_batch_times(self.logger, batch_times)
```

Comment:
I would move the call to `log_title` into `log_batch_times` itself, since we don't need to call `log_batch_times` without setting the title as well.